stories/message-box/message-box.stories.js

```diff
@@ -248,11 +254,12 @@ Information | \`fd-message-box--information\` | Information messages provide inf
 
 export const noIcon = () =>
     `
-<div class="fd-message-box-docs-static fd-message-box fd-message-box--no-icon fd-message-box--success fd-message-box--active">
+<div class="fd-message-box-docs-static fd-message-box fd-message-box--success fd-message-box--no-icon fd-message-box--active">
     <section class="fd-message-box__content">
         <header class="fd-bar fd-bar--header fd-message-box__header">
             <div class="fd-bar__left">
                 <div class="fd-bar__element">
+                    <i class="fd-message-box__title"></i>
```

Comment:
I don't think that `__title` is proper name for icon there, can you change it to basic `__icon`?

Reply:
fixed